Makefile.am

```diff
@@ -1009,6 +1009,8 @@ imap_libcyrus_imap_la_SOURCES = \
 	imap/conversations.h \
 	imap/convert_code.c \
 	imap/convert_code.h \
+	imap/css3_color.c \
+	imap/css3_color.h \
```

Comment:
Since imap/css3_color.h and imap/css3_color.c are linked into libcyrus_imap, `httpd`links with libcyrus_imap.so, the explicit dependency of css3_color.[ch] to `httpd` can be removed.

In any case, there is no point to mention the file `css3_color.h` twice in (different) `_SOURCE` variables.

Reply:
Fixed